solr/solr-ref-guide/src/schema-api.adoc

```diff
@@ -73,7 +73,7 @@ bin/solr -e cloud -noprompt
 
 == Modify the Schema
 
-To add, remove or replace fields, dynamic field rules, copy field rules, or new field types, you can send a POST request to the `/collection/schema/` endpoint with a sequence of commands in JSON format to perform the requested actions. The following commands are supported:
+To add, remove or replace fields, dynamic field rules, copy field rules, or new field types, you can send a POST request to the `/api/{collections|cores}/schema/` endpoint with a sequence of commands in JSON format to perform the requested actions. The following commands are supported:
```

Comment:
Should the path here be something like: `/api/{collections|cores}/{name}/schema`?  As-is I think it's missing the name of a specific collection.  It doesn't line up with the URL in the example below.

If I don't hear from you in a few days I'll make this change myself and merge.  
Thanks as always for the fixes @epugh